tsconfig.base.json

```diff
@@ -277,6 +277,96 @@
         "libs/gf/util-browser/src/index.ts"
       ],
       "@energinet-datahub/watt": ["libs/ui-watt/src/index.ts"],
+      "@energinet-datahub/watt/badge": [
```

Comment:
question: Are these path mapping entries maintained manually?

Reply:
Yes, they are; I have talked briefly with @ManBearTM that step 2 could be converting each part of Watt to being a lib on it’s own.

Reply:
We should adjust the `watt-component` generator to remove the line where `ui-watt/src/index.ts` file is updated.

We might also consider outputting a message in the console to remind us to update `tsconfig.base.json` file after generating a new Watt component.

Reply:
The generator has now been changed to update `tsconfig.base.json`